newproject01/newproject01/q1.c

```diff
@@ -0,0 +1,67 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#define N 8
```

Comment:
Choose a name that describes this const, `STRING_LENGTH` for example

Reply:
done